.github/workflows/delete-old-branches.yml

```diff
@@ -0,0 +1,20 @@
+name: Delete old branches
+on:
+  schedule:
+    - cron: "0 0 * * *"  # Runs every day at midnight (UTC)
+
+jobs:
+  delete-old-branches:
+    name: Delete old branches
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Run delete-old-branches-action
+        uses: beatlabs/delete-old-branches-action@v0.0.10
+        with:
+          repo_token: ${{ github.token }}
+          date: '7 days ago'
+          default_branches: main,master,staging
+          dry_run: true
```

Comment:
Once the PR is approved, I will remove this line, which will turn on the github action.